scripts/warnings-handling.py

```diff
@@ -0,0 +1,31 @@
+import os
+import re
+import sys
+
+
+warning_matcher = re.compile(r'.*warning[: ].*', re.I | re.S)
+warnings = 0
+
+def warnings_count():
+    global warnings
+    with open(os.path.join(os.getcwd(), 'build-log.txt'), 'r') as build_log:
```

Comment:
It make sense to replace fixed file name with command line parameter. The current value could be just default.

Reply:
Added `argparse` for it.